djconnectwise/admin.py

```diff
@@ -264,8 +264,8 @@ class ScheduleEntryInline(admin.StackedInline):
 @admin.register(models.Ticket)
 class TicketAdmin(admin.ModelAdmin):
     list_display = ('id', 'summary', 'status', 'record_type',)
-    list_filter = ('status', 'record_type')
-    search_fields = ['id', 'summary', 'company__name']
+    list_filter = ('record_type',)
+    search_fields = ['id', 'summary', 'company__name', 'status__name']
```

Comment:
The searching operation acts a little bit out of expectation. (see the captures below) Isn't that impossible to search for tickets using these 3 kinds of keywords?

![image](https://user-images.githubusercontent.com/73486314/118703689-3d529b00-b7cb-11eb-8ff6-6d7e6ae72017.png)

![image](https://user-images.githubusercontent.com/73486314/118703722-45aad600-b7cb-11eb-8b15-7fb4ba768671.png)

![image](https://user-images.githubusercontent.com/73486314/118703759-50fe0180-b7cb-11eb-8dd1-392118a79310.png)